### PR TITLE
Ergebnis amplifikation improvements

### PR DESCRIPTION
**Pull request by dd52**
see ticket https://redmine.healex.systems/issues/9121

Please check if fhirpath statements make sense

### CODE REVIEW

**Review by alexzautke — CHANGES_REQUESTED** [review]
Can you remove the compositions from this branch?
I would like to check and merge them separately.

**Comment by dd52** [comment]
done

File: Observation-ISH.StructureDefinition.json [hunk 1]
```diff
@@ -45,6 +45,18 @@
...
     "element": [
+      {
+        "id": "Observation",
+        "path": "Observation",
+        "constraint": [
+          {
+            "key": "amplifikation",
+            "severity": "error",
+            "human": "If Observation concerns MET, ALK or ROS1 then a different VS shall be bound",
+            "expression": "Observation.code.coding.code.where(code = ‘MET’) implies Observation.component.valueCodableConcept.coding.system.where(system = 'http://uk-koeln.de/fhir/ValueSet/nNGM/ergebnis-cish-met').exists() and Observation.code.coding.code.where(code = ‘MET’) implies Observation.component.valueCodableConcept.coding.system.where(system = ‘http://uk-koeln.de/fhir/ValueSet/amplification-met’).exists()"
```
Comment (alexzautke) [inline Observation-ISH.StructureDefinition.json]: Selecting polymorphic elements in FHIRPath is done differently, see http://hl7.org/fhirpath/N1/#paths-and-polymorphic-items

valueCodeableConcept is not allowed. ofType should be used.

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: Observation-ISH.StructureDefinition.json (hunk 1 above)
Comment (dd52) [reply Observation-ISH.StructureDefinition.json]: changed it to ofType

**Review by alexzautke — APPROVED** [review]

**Pull request merged**